modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/serieController.js

```diff
@@ -366,14 +378,15 @@ angular.module('adminNg.controllers')
     });
 
     $scope.themeSave = function () {
-        var theme = $scope.theme.current;
+        var selectedthemeid = $scope.selectedtheme.id;
+        $scope.updateSelectedThemeDescripton();
 
-        if (angular.isUndefined(theme) || theme === null) {
-            SeriesThemeResource.delete({id: $scope.resourceId}, {theme: theme}, function () {
+        if (angular.isUndefined(selectedthemeid) || selectedthemeid === null) {
+            SeriesThemeResource.delete({id: $scope.resourceId}, {theme: selectedthemeid}, function () {
```

Comment:
If you are touching these lines anyway, couldn't we add spaces inside the `{}` for consistency? Or is that inconsistent anyway? I saw it with spaces somewhere in this PR.